doc/de/Settings.md

```diff
@@ -367,27 +370,30 @@ Normalerweise trifft dies auf den ersten Account zu, der nach der Installation a
 Die Liste der E-Mail Adressen kann aber einfach erweitert werden.
 Mit keiner der angegebenen E-Mail Adressen können weitere Accounts registriert werden.
 
-    [config]
-    admin_email = you@example.com, buddy@example.com
+	'config' => [
```

Comment:
Here the indentation is wrong as well.

Reply:
I normalized both files' indentation to tabs.

Reply:
Thanks :-)